From: Yaroslav Halchenko <debian@onerussian.com>
Subject: BF: do not rely on identical FP results in a test_over_axis
  since numpy 1.7.0, for performance improvements, order of summation could be
different thus affecting precise FP result value. See 
http://permalink.gmane.org/gmane.comp.python.numeric.general/54061 for more
discussion/examples

--- a/mvpa2/tests/test_transformers.py
+++ b/mvpa2/tests/test_transformers.py
@@ -79,7 +79,7 @@ class TransformerTests(unittest.TestCase
         for axis in [None, 0, 1, 2]:
             oversum = OverAxis(np.sum, axis=axis)(data)
             sum_ = np.sum(data, axis=axis)
-            self.assertTrue(np.all(sum_ == oversum))
+            assert_array_almost_equal(sum_, oversum)
 
         # Transformer which doesn't modify dimensionality of the data
         data = data.reshape((6, -1))
